### PR TITLE
replace media rules and add css indentations

### PR DESCRIPTION
**Pull request by Olga1984**

### CODE REVIEW

File: client/src/searchPage/filterPanel/filterPanel.scss [hunk 1]
```diff
@@ -1,26 +1,24 @@
 .filter-wrapper {
+  max-width: 65rem;
+  width: 100%;
+  margin: 0 auto;
+  position: static;
+  @include breakpoint($wk-target-max-desktop) {
```
Comment (gshokanov) [inline client/src/searchPage/filterPanel/filterPanel.scss:7]: I guess there should be a new line before *include* as well. Please fix every other document if you don't object

**Review by gshokanov — CHANGES_REQUESTED** [review]

File: client/src/searchPage/filterPanel/filterPanel.scss (hunk 1 above)
Comment (Olga1984) [reply client/src/searchPage/filterPanel/filterPanel.scss:7]: Updated.

File: client/src/translate/translateController.js [hunk 2]
```diff
@@ -2,8 +2,6 @@ import angular from 'angular';
 
 export default angular.module('translate')
     .controller('TranslateController', ['$translate', function ($translate) {
-        this.selectedLanguage = $translate.use();
-
         this.changeLanguage = (key) => {
             $translate.use(key);
             this.selectedLanguage = key;
```
Comment (gshokanov) [inline client/src/translate/translateController.js]: I guess you should remove this line as well?

**Review by gshokanov — CHANGES_REQUESTED** [review]

File: client/src/translate/translateController.js (hunk 2 above)
Comment (Olga1984) [reply client/src/translate/translateController.js]: Updated.

**Review by gshokanov — APPROVED** [review]

**Pull request merged**